app/talk/data-export-button.cjsx

```diff
@@ -10,11 +9,14 @@ module.exports = React.createClass
     isAvailable: null
     exportRequested: false
     exportError: null
+    mostRecent: null
 
   exportGet: ->
     @_exportsGet or= talkClient.type('data_requests').get(section: @section(), kind: @props.exportType).then (requests) =>
-      @setState(isAvailable: true) if requests.length > 0
-      requests
+      if requests.length > 0
```

Comment:
I don't think this conditional is working as intended. When I click the data export button for comments, I see in the network tab that the POST request is going out and there is a response, but the UI isn't changing from showing "no recent requests"